### PR TITLE
Update index documentation

### PR DESCRIPTION
**Pull request by miquelbar**

### CODE REVIEW

**Review by treezio — COMMENTED** [review]
Shouldn't we add `terraform import` documentation in
https://github.com/sequra/terraform-provider-honeybadger/blob/master/docs/resources/user.md
for users and also
https://github.com/sequra/terraform-provider-honeybadger/blob/master/docs/resources/team.md
for teams?

I guess https://github.com/sequra/terraform-provider-honeybadger/blob/master/docs/index.md documentation is related to the provider artifact itself rather than the resources it manages

**Comment by miquelbar** [comment]
> Shouldn't we add `terraform import` documentation in https://github.com/sequra/terraform-provider-honeybadger/blob/master/docs/resources/user.md for users and also https://github.com/sequra/terraform-provider-honeybadger/blob/master/docs/resources/team.md for teams?
> 
> I guess https://github.com/sequra/terraform-provider-honeybadger/blob/master/docs/index.md documentation is related to the provider artifact itself rather than the resources it manages

Good point! I'll try to do it 😄

File: docs/resources/team.md [hunk 1]
```diff
@@ -1,16 +1,24 @@
+
 ---
-# generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "honeybadger_team Resource - terraform-provider-honeybadger"
-subcategory: ""
+layout: ""
+page_title: "HoneyBadger: honeybadger_team"
 description: |-
-  
+  Creates and manages teams within your HoneyBadger organization
```
Comment (treezio) [inline docs/resources/team.md]:
```suggestion
  Creates and manages teams within your Honeybadger organization
```
🙈

File: docs/resources/team.md [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
 ---
 
-# honeybadger_team (Resource)
+# honeybadger_user
 
+This resource allows you to create and manage teams within your HoneyBadger organization.
```
Comment (treezio) [inline docs/resources/team.md]:
```suggestion
This resource allows you to create and manage teams within your Honeybadger organization.
```
🙈

File: docs/resources/team.md [hunk 3]
```diff
@@ -1,16 +1,24 @@
+
 ---
-# generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "honeybadger_team Resource - terraform-provider-honeybadger"
-subcategory: ""
+layout: ""
+page_title: "HoneyBadger: honeybadger_team"
```
Comment (treezio) [inline docs/resources/team.md]:
```suggestion
page_title: "Honeybadger: honeybadger_team"
```
🙈

File: docs/resources/user.md [hunk 4]
```diff
@@ -1,16 +1,28 @@
+
 ---
-# generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "honeybadger_user Resource - terraform-provider-honeybadger"
-subcategory: ""
+layout: ""
+page_title: "HoneyBadger: honeybadger_user"
```
Comment (treezio) [inline docs/resources/user.md]:
```suggestion
page_title: "Honeybadger: honeybadger_user"
```
🙈

File: docs/resources/user.md [hunk 5]
```diff
@@ -1,16 +1,28 @@
...
 description: |-
-  
+  Creates and manages users within HoneyBadger organizations
```
Comment (treezio) [inline docs/resources/user.md]:
```suggestion
  Creates and manages users within Honeybadger organizations
```
🙈

File: docs/resources/user.md [hunk 6]
```diff
@@ -1,16 +1,28 @@
...
 ---
 
-# honeybadger_user (Resource)
+# honeybadger_user
 
+This resource allows you to create and manage users within your HoneyBadger organization.
```
Comment (treezio) [inline docs/resources/user.md]:
```suggestion
This resource allows you to create and manage users within your Honeybadger organization.
```
🙈

File: templates/resources/team.md.tmpl [hunk 7]
```diff
@@ -0,0 +1,27 @@
+
+---
+layout: ""
+page_title: "HoneyBadger: honeybadger_team"
```
Comment (treezio) [inline templates/resources/team.md.tmpl]:
```suggestion
page_title: "Honeybadger: honeybadger_team"
```
🙈

File: templates/resources/team.md.tmpl [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+description: |-
+  Creates and manages teams within your HoneyBadger organization
```
Comment (treezio) [inline templates/resources/team.md.tmpl]:
```suggestion
  Creates and manages teams within your Honeybadger organization
```
🙈

File: templates/resources/team.md.tmpl [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+# honeybadger_user
+
+This resource allows you to create and manage teams within your HoneyBadger organization.
```
Comment (treezio) [inline templates/resources/team.md.tmpl]:
```suggestion
This resource allows you to create and manage teams within your Honeybadger organization.
```
🙈

File: templates/resources/user.md.tmpl [hunk 10]
```diff
@@ -0,0 +1,27 @@
+
+---
+layout: ""
+page_title: "HoneyBadger: honeybadger_user"
+description: |-
+  Creates and manages users withi your HoneyBadger organization
```
Comment (treezio) [inline templates/resources/user.md.tmpl]:
```suggestion
  Creates and manages users withi your Honeybadger organization
```
🙈

File: templates/resources/user.md.tmpl [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+# honeybadger_user
+
+This resource allows you to create and manage users within your HoneyBadger organization.
```
Comment (treezio) [inline templates/resources/user.md.tmpl]:
```suggestion
This resource allows you to create and manage users within your Honeybadger organization.
```
🙈

File: templates/resources/user.md.tmpl [hunk 12]
```diff
@@ -0,0 +1,27 @@
+
+---
+layout: ""
+page_title: "HoneyBadger: honeybadger_user"
```
Comment (treezio) [inline templates/resources/user.md.tmpl]:
```suggestion
page_title: "Honeybadger: honeybadger_user"
```
🙈

**Review by treezio — COMMENTED** [review]
Sorry for the monkey gang 🙈

**Review by treezio — APPROVED** [review]
Thanks! 🚀 🚀 🚀

**Pull request merged**